### ISSUE TITLE
fill() prints histogram in 0.10.1

### ISSUE THREAD

**Issue by alexander-held** [scikit-hep/boost-histogram#437]
Hi, since version 0.10.1, calling a `.fill()` command on a `boost_histogram.Histogram` prints the histogram:

```
import boost_histogram as bh

hist = bh.Histogram(bh.axis.Variable([1,2]))
hist.fill([1])
```
prints
```
(array([1]),) None None
```
while in version 0.10.0 there is no output.

---

environment: running via `python:3.8-slim bash` via docker:
```
boost-histogram==0.10.1
numpy==1.19.1
```

**Comment by henryiii** [comment]
Not again! I left a debug output in 0.7.0 too. I will see if I can set up a pre-commit check that looks for print statements.

**Comment by henryiii** [comment]
Here it is:

https://github.com/scikit-hep/boost-histogram/blob/e79ff55a2ca0be241dd508e55f78b26e1d881867/src/boost_histogram/_internal/hist.py#L295

I ignored the print output last time, because it was in a lesser-used function, but this one probably needs a patch sooner rather than later.

**Comment by henryiii** [comment]
(This has been fixed in 0.10.2)
